runtime/testy/stateManager.py

```diff
@@ -32,12 +32,52 @@ def addPipe(self, processName, pipe):
     self.processMapping[processName] = pipe
     pipe.send(RUNTIME_CONFIG.PIPE_READY.value)
 
-  def getValue(self, key):
-    self.processMapping[PROCESS_NAMES.STUDENT_CODE].send(self.state[key])
+  def getValue(self, keys):
+    result = self.state
+    i = 0
```

Comment:
I just noticed your comment about the enumerate function.  Did not know that that function existed.  I'll try it out